.github/workflows/ci.yml

```diff
@@ -49,6 +49,10 @@ jobs:
     - name: Run twine checker
       run: |
         twine check dist/*
+    - name: Run mypy
+      run: |
+        pip install -r tests/requirements-mypy.txt
+        mypy aioredis --ignore-missing-imports
```

Comment:
It's better to just run `mypy` with no arguments, and keep all configuration in `.mypy.ini`.

Good example of a config from one of our other repos:
https://github.com/aio-libs/aiohttp-jinja2/blob/master/.mypy.ini

Reply:
Ah just copied our makefile; good catch since we have all dependencies installed. Not sure of the reasoning behind the flag to begin with though.

Reply:
But, even the `aioredis` can be removed and just included in the config file as `files = aioredis`.

We'll also want mypy to run on tests/examples at some point as well. We've had bugs with type annotations which are only caught if you type check actual usage (plus it keeps the examples up-to-date with the library code).

Reply:
We don't have a config file, so updated in CI only. Maybe in the future.